### PR TITLE
Support Stylelint 15

### PR DESCRIPTION
**Pull request by fregante**
Tentative PR to support the new version. I don't know if anything else is necessary

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: stylelint-declaration-block-no-ignored-properties@2.6.0
npm WARN Found: stylelint@15.0.0
npm WARN node_modules/stylelint
npm WARN   peer stylelint@">=14" from stylelint-config-xo@0.21.1
npm WARN   node_modules/stylelint-config-xo
npm WARN     dev stylelint-config-xo@"^0.21.1" from the root project
npm WARN   1 more (the root project)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer stylelint@"^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0" from stylelint-declaration-block-no-ignored-properties@2.6.0
npm WARN node_modules/stylelint-config-xo/node_modules/stylelint-declaration-block-no-ignored-properties
npm WARN   stylelint-declaration-block-no-ignored-properties@"^2.5.0" from stylelint-config-xo@0.21.1
npm WARN   node_modules/stylelint-config-xo
npm WARN 
npm WARN Conflicting peer dependency: stylelint@14.16.1
npm WARN node_modules/stylelint
npm WARN   peer stylelint@"^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0" from stylelint-declaration-block-no-ignored-properties@2.6.0
npm WARN   node_modules/stylelint-config-xo/node_modules/stylelint-declaration-block-no-ignored-properties
npm WARN     stylelint-declaration-block-no-ignored-properties@"^2.5.0" from stylelint-config-xo@0.21.1
npm WARN     node_modules/stylelint-config-xo
```
```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: stylelint-order@5.0.0
npm WARN Found: stylelint@15.0.0
npm WARN node_modules/stylelint
npm WARN   peer stylelint@">=14" from stylelint-config-xo@0.21.1
npm WARN   node_modules/stylelint-config-xo
npm WARN     dev stylelint-config-xo@"^0.21.1" from the root project
npm WARN   1 more (the root project)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer stylelint@"^14.0.0" from stylelint-order@5.0.0
npm WARN node_modules/stylelint-config-xo/node_modules/stylelint-order
npm WARN   stylelint-order@"^5.0.0" from stylelint-config-xo@0.21.1
npm WARN   node_modules/stylelint-config-xo
npm WARN 
npm WARN Conflicting peer dependency: stylelint@14.16.1
npm WARN node_modules/stylelint
npm WARN   peer stylelint@"^14.0.0" from stylelint-order@5.0.0
npm WARN   node_modules/stylelint-config-xo/node_modules/stylelint-order
npm WARN     stylelint-order@"^5.0.0" from stylelint-config-xo@0.21.1
npm WARN     node_modules/stylelint-config-xo

```

### CODE REVIEW

**Comment by fregante** [comment]
I think that the issue isn't even here, actually 🤔 😅

**Comment by fregante** [comment]
This is needed here though https://github.com/hudochenkov/stylelint-order/releases/tag/6.0.2

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -10,12 +10,11 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 16
-          - 14
-          - 12
+          - 20
+          - 18
```
Comment (fregante) [inline .github/workflows/main.yml:14]: Node 18 is already LTS: https://nodejs.org/en

Comment (sindresorhus) [reply .github/workflows/main.yml:14]: But 16 is also still LTS, and I generally prefer to support all active LTS versions.

**Pull request merged**